docs/content/en/docs/clustermgmt/observability/expose-metrics.md

```diff
@@ -0,0 +1,146 @@
+    ```bash
+    cat << EOF | kubectl apply -f -
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: metrics-reader
+    rules:
+      - nonResourceURLs:
+          - "/metrics"
+        verbs:
+          - get
+    EOF
+    ```
+
+2.  Create a new cluster role binding to bind the above cluster role to the client pod's service account.
+    
+    This is an example cluster role binding for the prometheus client.
+    ```bash
+    cat << EOF | kubectl apply -f -
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: metrics-reader-binding
+    subjects:
+    - kind: ServiceAccount
+      name: prometheus-server
+      namespace: observability
+    roleRef:
+      kind: ClusterRole
+      name: metrics-reader
+      apiGroup: rbac.authorization.k8s.io
+    EOF
+    ```
+
+3.  Create a config map to store the proxy configuration.
+    
+    This is an example config map for HAProxy.
+    ```bash
+    cat << EOF | kubectl apply -f -
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: metrics-proxy
+    data:
+      haproxy.cfg: |
+        defaults
+          mode http
+          timeout connect 5000ms
+          timeout client 5000ms
+          timeout server 5000ms
+          default-server maxconn 10
+
+        frontend kube-proxy
+          bind ${NODE_IP}:10249
+          http-request deny if !{ path /metrics }
+          default_backend kube-proxy
+        backend kube-proxy
+          server kube-proxy 127.0.0.1:10249 check
+
+        frontend kube-controller-manager
+          bind ${NODE_IP}:10257
+          http-request deny if !{ path /metrics }
+          default_backend kube-controller-manager
+        backend kube-controller-manager
+          server kube-controller-manager 127.0.0.1:10257 ssl verify none check
+
+        frontend kube-scheduler
+          bind ${NODE_IP}:10259
+          http-request deny if !{ path /metrics }
+          default_backend kube-scheduler
+        backend kube-scheduler
+          server kube-scheduler 127.0.0.1:10259 ssl verify none check
+    EOF
+    ```
+
+4.  Create a daemonset for the proxy and mount the config map volume onto the proxy pods.
+    
+    This is an example configuration for the HAProxy daemonset.
+    ```bash
+    cat << EOF | kubectl apply -f -
+    apiVersion: apps/v1
+    kind: DaemonSet
+    metadata:
+      name: metrics-proxy
+    spec:
+      selector:
+        matchLabels:
+          app: metrics-proxy
+      template:
+        metadata:
+          labels:
+            app: metrics-proxy
+        spec:
+          tolerations:
+          - key: node-role.kubernetes.io/control-plane
+            operator: Exists
+            effect: NoSchedule
+          hostNetwork: true
+          containers:
+            - name: haproxy
+              image: public.ecr.aws/docker/library/haproxy:latest
```

Comment:
lets use the eksa built image: `public.ecr.aws/eks-anywhere/kubernetes-sigs/kind/haproxy:v0.20.0-eks-a-54`